.github/workflows/ci.yml

```diff
@@ -12,16 +12,15 @@ jobs:
       - name: Checkout
         uses: actions/checkout@v2
       - name: Install Ruby
-        uses: eregon/use-ruby-action@v1.10.0
+        uses: eregon/use-ruby-action@v1
         with:
           ruby-version: ${{ matrix.ruby }}
       - name: Install dependencies
         run: |
-          gem install -N bundler
-          # NOTE using bundle config causes bundle install to crash; see https://github.com/eregon/use-ruby-action/issues/13
-          #bundle config --local path .bundle/gems
-          #bundle config --local without docs lint coverage
-          #bundle --jobs 3 --retry 3
-          bundle --jobs 3 --retry 3 --path .bundle/gems --without docs lint coverage
+          bundle config --local path .bundle/gems
+          bundle config --local without docs lint coverage
+          # warning output can confuse Powershell in CI, (multi-line scripts & exit codes)
+          $env:RUBYOPT = '-W0'
```

Comment:
Sweet trick!

Reply:
Except don't we need to unset it afterwards so it doesn't remain set while running tests?

Reply:
> Except don't we need to unset it afterwards so it doesn't remain set while running tests?

No.  Big difference between Azure Pipelines / GitHub Actions and Travis / AppVeyor.  In Actions, changes to the environment do not affect subsequent steps, unless one uses something like:
```
echo '::set-env name=JOBS::'-j$((1 + $(nproc --all)))
```

Sometimes it's helpful, often not...

Reply:
Thanks for the info!